hbase_sql/src/main/resources/yarn-site.xml

```diff
@@ -105,12 +105,12 @@
   </property>
   <property>
     <name>yarn.nodemanager.resource.memory-mb</name>
-    <value>4096</value>
+    <value>8192</value>
```

Comment:
Are we requiring more memory than before now?

Reply:
On standalone we are starting one more process so we do require more memory. We won't be using the full 8Gb though, that's an upper limit for reservations, which could be made tighter if we wanted.